openxcframework/VehicleManager.swift

```diff
@@ -53,26 +50,12 @@ public enum VehicleManagerStatusMessage: Int {
 // This enum is outside of the main class for ease of use in the client app. It allows
 // for referencing the enum without the class hierarchy in front of it. Ie. the enums
 // can be accessed directly as .C5DETECTED for example
-
-
 // public enum VehicleManagerConnectionState
 // values reported in public variable connectionState
-/*public enum VehicleManagerConnectionState: Int {
-  case notConnected=0           // not connected to any C5 VI
-  case scanning=1               // VM is allocation and scanning for nearby VIs
-  case connectionInProgress=2   // connection in progress (connecting/searching for services)
-  case connected=3              // connection established (but not ready to receive btle writes)
-  case operational=4            // C5 VI operational (notify enabled and writes accepted)
-}*/
 // This enum is outside of the main class for ease of use in the client app. It allows
```

Comment:
This 3 line comment is exactly the same as the one 2 lines above it. This can be removed.

Reply:
deleted the lines.